lib/ansible/modules/packaging/os/snap.py

```diff
@@ -108,7 +107,8 @@ def refresh_snap(module):
 def install_snap(module, snap_name):
```

Comment:
Can you accept a list here as well? And then use whole bunch of packages in install command?

Reply:
Since I loop over the packages to check if they exit *before*, I can indeed install all the packages in one go here. I will update it.

Reply:
Mmh. I call is_snap_installed though.

Reply:
Here is how I did it: https://github.com/ansible/ansible/pull/44939/commits/8e0073bc63103de46d7fdb15e0a6895967197f7e